README.md

```diff
@@ -1,28 +1,72 @@
+* [Development server](#development-server)
+* [Code scaffolding](#code-scaffolding)
+* [Build](#build)
+* [Tests](#tests)
+* [Understand your workspace](#understand-your-workspace)
+* [Running with docker](#running-with-docker)
+* [Package as a runnable Docker container](#package-as-a-runnable-docker-container)
+* [Further help](#further-help)
+
+
+
+
+
+## What is the app for
+- This project is a reference implementation to showcase best practices to use when building a new Angular SPA for Backbase components and libraries. The project is updated with the latest versions of the tools and libraries used. It shows guidelines about how to develop a web application for the frontend developers.
+- This prototype includes the following:
+    - Authentication
+    - Support of the multiple languages
+    - Integration with a theme
+    - Using existing journey installed from local NPM repository
+    - Configuring journeys
+    - Configuring entitlements
+    - Communication between journeys
+ 
+ ### What components are part of the app
+  - Auth module
+  - Locale selector for SPA
+  - Theme
+  - Entitlement
+  - Configuration of journeys
+  - Communication service
+  - Simplistic journey examples like transactions and transfer
+
+## Prerequisites
+For the best experience, the following VSCode extensions for Nx and Jest should be installed. 
+
+- [nrwl.angular-console](https://marketplace.visualstudio.com/items?itemName=nrwl.angular-console): to find and run all the possible Nx Commands.J
+- [firsttris.vscode-jest-runner](https://marketplace.visualstudio.com/items?itemName=firsttris.vscode-jest-runner): to easily run the isolated test as you are developing. 
+
+
+- On some AWS environments, due to specific WAF configuration, you may haved to use `http://0.0.0.0:4200/` when accessing the app locally, in order to successfully authenticate.
+
 
-On some AWS environments, due to specific WAF configuration, you might need to use `http://0.0.0.0:4200/` when accessing the app locally, in order to successfully authenticate.
 
 ## Authentication details
 
-### Prerequisites
+#### Prerequisites
 
 - Go through the [documentation page](https://community.backbase.com/documentation/foundation_angular/latest/authenticate_users) first to avoid any confusion with implementing authentication in the modelless app
 
-### Important Info
+#### Important Info
 
 The code that is used for authentication is not for production purposes, this is the example to understand the concepts lying under the hood.
 Do not copy-paste anything related to the authentication to your banking application.
 
-### How to add authentication to your app
+#### How to add authentication to your app
 
-See the example code in the `app.module.ts`, the related `AuthConfig` in the `environment.ts` files, and the `APP_INITIALIZER` provider logic.
+Check the example code in the `app.module.ts`, the related `AuthConfig` in the `environment.ts` files, and the `APP_INITIALIZER` provider logic.
```

Comment:
Can you not add a direct link to the file, and maybe to the lines you want the developer to pay attention to?

Reply:
Agree, See my comment above